init documentation for `ServiceWorkerGlobalScope.cookiechange_event` (#30933)
mdn/content: files/en-us/web/api/serviceworkerglobalscope/index.md

  - : Fired on a payment app's service worker to check whether it is ready to handle a payment. Specifically, it is fired when the merchant website calls {{domxref("PaymentRequest.PaymentRequest", "new PaymentRequest()")}}.
- {{domxref("ServiceWorkerGlobalScope/contentdelete_event", "contentdelete")}} {{Experimental_Inline}}
  - : Occurs when an item is removed from the {{domxref("ContentIndex", "Content Index")}}.
- {{domxref("ServiceWorkerGlobalScope/cookiechange_event", "cookiechange")}} {{Experimental_Inline}}

>   - : Fired when any cookie changes have occurred which match the Service Worker's cookie change subscription list.

Reviewer: ```suggestion
  - : Fired when a cookie change has occurred that matches the service worker's cookie change subscription list.
```
